app/controllers/owner/vans_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Owner::VansController < ApplicationController
+  def new
+    @van = Van.new
+  end
+
+  def create
+    van = Van.new(vans_params)
+    van.owner = current_user
+    van.save
+
+    redirect_to vans_path
```

Comment:
It's redirected to awner_dashborad_path instead of vans_path. It allows the owner to see the lists of his vans on the dashboard after creating it